test/unit/detail/version_check_test.hpp

```diff
@@ -213,6 +213,20 @@ TEST_F(version_check, option_on)
     if (app_call_succeeded)
     {
         EXPECT_TRUE(std::filesystem::exists(app_version_filename()));
+
+        // This is not an actual check s.t. the test does not fail because there is something wrong with
+        // the server in Tuebingen. But upon manual execution of the test this can give valuable insight
+        // on whats happening.
```

Comment:
Can you rephrase that?
I don't understand it.

Do you mean that this test succeeds even if the server is down?
`manual execution` as opposed to what? 
And what kind of insight you may get?
It seems more like debug output, not like a log of what was happening.